src/main/java/com/gregtechceu/gtceu/api/machine/IMachineBlockEntity.java

```diff
@@ -53,7 +54,14 @@ default void scheduleRenderUpdate() {
     }
 
     default long getOffsetTimer() {
-        return level() == null ? getOffset() : (level().getServer().getTickCount() + getOffset());
+        if (level() == null) {
+            return getOffset();
+        } else if (level().isClientSide()) {
+            return GTValues.CLIENT_TIME + getOffset();
+        } else if (level().getServer() == null) {
```

Comment:
Does this want to be `!= null`?

Reply:
yes, when I split the middle condition I forgot to invert this part